cupyx/scipy/ndimage/measurements.py

```diff
@@ -219,8 +219,8 @@ def _kernel_finalize():
 
 
 _ndimage_sum_kernel = cupy.ElementwiseKernel(
-    'T input, R labels, raw X index, int32 size',
-    'raw Y out',
+    'T input, R labels, raw X index, uint64 size',
+    'raw float64 out',
```

Comment:
Would you use the output dtype of `uint64` for integer inputs, as well as the same dtype with `input` for floating-point inputs as `scipy.ndimage.sum` does so.

Reply:
So i just want to confirm what I understand from this, it should be something like
```python
if issubtype(input.dtype, cupy.integer):
    out_type = cupy.uint64
else:
    out_type = input.dtype
```
because the following code 
```python
import numpy as np
import scipy.ndimage as ndn
input = np.array([1, 2, 3])
labels = np.array([1, 1, 0])
index = np.array([1, 0])
assert(input.dtype == np.int64)
assert(labels.dtype == np.int64)
assert(labels.dtype == np.int64)
out = ndn.sum(input, labels, index)
print(out.dtype)

```
gives

> float64
And from what I understood atleast this will be non-compatible. Please correct me if I misunderstood your request.

Reply:
Thanks to confirm. You're right, and what I saw might be the output dtype of `ndn.sum(input)`. Sorry, you don't need to make any change.